### PR TITLE
[ISSUE #69] support will feature.

### LINKED ISSUE #69: To support Will Message Feature
**Will Message** is MQTT's ability to gracefully send wills to third parties for devices that may be unexpectedly disconnected. The Current version supports the MQTT 3.1.1 protocol, but the **Will Message** feature in this version is not yet supported. So we are discussing plans to support this feature.

### PR DESCRIPTION
**Pull request by scutcr7**
send messages when the server is down using scheduled tasks to close #69

### CODE REVIEW

**Comment by pingww** [comment]
add some unit tests

**Comment by DongyuanPan** [comment]
The KV will be removed and the code can be submitted to the Develop branch based on the state machine

File: mqtt-cs/pom.xml [hunk 1]
```diff
@@ -70,6 +74,12 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.rocketmq</groupId>
+            <artifactId>meta</artifactId>
```
Comment (ShannonDing) [inline mqtt-cs/pom.xml:79]: where is this packeage.

Comment (DongyuanPan) [reply mqtt-cs/pom.xml:79]: meta in master branch. It has been removed in develop branch

**Comment by DongyuanPan** [comment]
This PR can be closed, see #174

**Pull request merged**